src/videojs-http-streaming.js

```diff
@@ -617,6 +618,20 @@ class VhsHandler extends Component {
     });
 
     this.on(this.tech_, 'play', this.play);
+
+    if (this.options_.useNetworkInformation) {
+      this.networkInformation = window.navigator.connection || window.navigator.mozConnection || window.navigator.webkitConnection;
+
+      if (this.networkInformation) {
+        this.networkInformation.addEventListener('change', () => this.setNetworkInformationStats_());
```

Comment:
Since `downlink` is a property on `navigator.connection`, I think we may be better off accessing the property instead of adding a new property and updating it on each `change`.

Reply:
Agree. Was originally planning on storing more properties to support future use-cases with this API, but no need to get ahead of ourselves.